executor/executor_bsd.h

```diff
@@ -139,9 +140,13 @@ static void cover_protect(cover_t* cov)
 
 static void cover_unprotect(cover_t* cov)
 {
+	if (cov->data == NULL)
+		fail("cover_unprotect invoked on an unmapped cover_t object");
 #if GOOS_freebsd
-	size_t mmap_alloc_size = kCoverSize * KCOV_ENTRY_SIZE;
-	mprotect(cov->data, mmap_alloc_size, PROT_READ | PROT_WRITE);
+	if (cov->data) {
```

Comment:
You don't need to check for `if (cov->data)` again.

Reply:
Oops, thanks, fixed.